stdlib/public/core/Integers.swift.gyb

```diff
@@ -14,6 +14,8 @@
 # Utility code for later in this template
 #
 
+from SwiftIntTypes import all_integer_types, int_max_bits, should_define_truncating_bit_pattern_init
```

Comment:
Are we keeping the `truncating:` init?  Since we have `truncatingOrExtending:` it seems slightly unnecessary.  Though it could help expressivity...